docs/topics/mta-import-questionnaire.adoc

```diff
@@ -5,19 +5,21 @@
 
 :_content-type: PROCEDURE
 [id="mta-import-questionnaire_{context}"]
-= Importing questionnaire
+= Importing an assessment questionnaire
+
+In the {ProductName} ({ProductShortName}) 7.0, you can upload an assessment questionnaire from your system. 
+
+WARNING: The name of the imported questionnaire must be unique.
+If the name, which is defined in the YAML syntax (`name:<name of questionnaire>`), is duplicated, the import will fail with the following error message: `UNIQUE constraint failed: Questionnaire.Name`.
+
 
 .Procedure
 
 . In the *Administration* view, select *Assessment questionnaires*.
-. To import a questionnaire, click *Import questionnaire*.
+. Click *Import questionnaire*.
 . Click *Upload*.
-. Navigate to the location of your questionnaire, and click *Open*.
-. Click *Import*.
-
-[WARNING]
-.UNIQUE constraint failed: Questionnaire.Name
-====
-The name of the imported questionnaire must be unique.
-If the name, which is defined in the YAML syntax (`name:<name of questionnaire>`), is duplicated, the import will fail and an error is returned `UNIQUE constraint failed: Questionnaire.Name`. The name of the YAML file is irrelevant.
-====
```

Comment:
Please, can we have this `WARNING` included somewhere? I could not see it included anywhere else
Thanks

Reply:
Sorry found it... please ignore